pkg/dynchan/README.md

```diff
@@ -0,0 +1,47 @@
+
+# Dynchan
+
+## Go code
+
+```go
+
+inputCh, outputCh := dynchan.PipeChan(1000)
+
+
+// Send any number of objects without being blocked
+for i := 0; i < 1*1000*1000 ; i++ {
+    inputCh <- "msg" 
+}
+
+
+// Receive and process objects
+for m := range outputCh {
+    // ...
+}
+
+
+```
```

Comment:
Can you show an example of integrated work like, for example use this in `client-fs.go` for events..